pkg/pki/sshkey.go

```diff
@@ -73,13 +73,19 @@ func ComputeAWSKeyFingerprint(publicKey string) (string, error) {
 		return "", err
 	}
 
-	der, err := toDER(sshPublicKey)
-	if err != nil {
-		return "", fmt.Errorf("error computing fingerprint for SSH public key: %v", err)
+	switch fmt.Sprintf("%T", sshPublicKey) {
+	case "*ssh.rsaPublicKey":
+		der, err := rsaToDER(sshPublicKey)
+		if err != nil {
+			return "", fmt.Errorf("error computing fingerprint for SSH public key: %v", err)
+		}
+		h := md5.Sum(der)
+		return colonSeparatedHex(h[:]), nil
+	case "ssh.ed25519PublicKey":
+		return strings.TrimPrefix(ssh.FingerprintSHA256(sshPublicKey), "SHA256:"), nil
```

Comment:
Is there a reason for using SHA256 for ed keys, but MD5 for RSA?
Perhaps use MD5 for both, optionally add a flag for hash type (like `ssh-add -l` does).

Reply:
Just following the same method the AWS API use to calculate the fingerprint. I added an example in the PR description 😃

Reply:
Fair enough. I think we should keep the prefix so we know what hash algo is actually used.